### ISSUE TITLE
pre-trained model weights

### ISSUE THREAD

**Issue by asdhajksdh** [JusperLee/TIGER#12]
Hello, I couldn't find the pre trained weights for your post.

**Comment by asdhajksdh** [comment]
Trainable params: 821 K                                                                                                                                                                    
Non-trainable params: 0                                                                                                                                                                    
Total params: 821 K                                                                                                                                                                        
Total estimated model params size (MB): 3                                                                                                                                                  
Epoch 0/499 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5067/5067 1:19:07 • 0:00:00 1.07it/s v_num: eye7 train_loss_step: -1.973 Metric val_loss/dataloader_idx_0 improved. New best score: -2.711
Epoch 0, global step 5067: 'val_loss/dataloader_idx_0' reached -2.71084 (best -2.71084), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet/epoch=0.ckpt' as top 5
Epoch 1/499 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5067/5067 1:18:47 • 0:00:00 1.06it/s v_num: eye7 train_loss_step: -5.716 val_loss/dataloader_idx_0: -2.711 test_loss/dataloader_idx_1: -2.609 
                                                                                          lr: 0.001 train_loss_epoch: -3.752                                                                       Metric val_loss/dataloader_idx_0 improved by 1.595 >= min_delta = 0.0. New best score: -4.305
Epoch 2/499 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5067/5067 1:18:41 • 0:00:00 1.07it/s v_num: eye7 train_loss_step: -6.641 val_loss/dataloader_idx_0: -4.305 test_loss/dataloader_idx_1: -2.609 
                                                                                          lr: 0.001 train_loss_epoch: -4.86                                                                        Metric val_loss/dataloader_idx_0 improved by 1.183 >= min_delta = 0.0. New best score: -5.489
Epoch 3/499 ━━━━━━━━━━━╺━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1423/5067 0:22:03 • 0:56:21 1.08it/s v_num: eye7 train_loss_step: -7.168 val_loss/dataloader_idx_0: -5.489 test_loss/dataloader_idx_1: -2.609 
                                                                                          lr: 0.001 train_loss_epoch: -5.643           我的看着好像不正常啊

**Comment by asdhajksdh** [comment]
python audio_train.py --conf_dir configs/tiger.yml  这是从头训练的吧，我好像没有看到加载你huggingface上的权重，这个loss 是不是有些奇怪？

**Comment by JusperLee** [comment]
你的数据集是什么？

**Comment by asdhajksdh** [comment]
是在您公开的EchoSet数据集，我只改动了gpu和batch size的设置，单卡，batch为 2进行的训练 

这是tiger-small的
Epoch 0, global step 10134: 'val_loss/dataloader_idx_0' reached -2.81319 (best -2.81319), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_samll_4.30/epoch=0.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 1.856 >= min_delta = 0.0. New best score: -4.669
Epoch 1, global step 20268: 'val_loss/dataloader_idx_0' reached -4.66917 (best -4.66917), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_samll_4.30/epoch=1.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 1.058 >= min_delta = 0.0. New best score: -5.728
Epoch 2, global step 30402: 'val_loss/dataloader_idx_0' reached -5.72760 (best -5.72760), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_samll_4.30/epoch=2.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 0.909 >= min_delta = 0.0. New best score: -6.637
Epoch 3, global step 40536: 'val_loss/dataloader_idx_0' reached -6.63651 (best -6.63651), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_samll_4.30/epoch=3.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 1.382 >= min_delta = 0.0. New best score: -8.019
Epoch 4, global step 50670: 'val_loss/dataloader_idx_0' reached -8.01901 (best -8.01901), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_samll_4.30/epoch=4.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 1.130 >= min_delta = 0.0. New best score: -9.149
Epoch 5, global step 60804: 'val_loss/dataloader_idx_0' reached -9.14938 (best -9.14938), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_samll_4.30/epoch=5.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 0.401 >= min_delta = 0.0. New best score: -9.551
Epoch 6, global step 70938: 'val_loss/dataloader_idx_0' reached -9.55069 (best -9.55069), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_samll_4.30/epoch=6.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 0.101 >= min_delta = 0.0. New best score: -9.652
Epoch 7, global step 81072: 'val_loss/dataloader_idx_0' reached -9.65178 (best -9.65178), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_samll_4.30/epoch=7.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 0.557 >= min_delta = 0.0. New best score: -10.209
Epoch 8, global step 91206: 'val_loss/dataloader_idx_0' reached -10.20859 (best -10.20859), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_samll_4.30/epoch=8.ckpt' as top 5

这是tiger-large
Epoch 0, global step 10134: 'val_loss/dataloader_idx_0' reached -2.71446 (best -2.71446), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_4.30/epoch=0.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 2.455 >= min_delta = 0.0. New best score: -5.170
Epoch 1, global step 20268: 'val_loss/dataloader_idx_0' reached -5.16980 (best -5.16980), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_4.30/epoch=1.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 1.852 >= min_delta = 0.0. New best score: -7.022
Epoch 2, global step 30402: 'val_loss/dataloader_idx_0' reached -7.02223 (best -7.02223), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_4.30/epoch=2.ckpt' as top 5
Epoch 3, global step 40536: 'val_loss/dataloader_idx_0' reached -6.75066 (best -7.02223), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_4.30/epoch=3.ckpt' as top 5
Metric val_loss/dataloader_idx_0 improved by 1.518 >= min_delta = 0.0. New best score: -8.540
Epoch 4, global step 50670: 'val_loss/dataloader_idx_0' reached -8.53997 (best -8.53997), saving model to '/data/private/TIGER/Experiments/checkpoint/TIGER-EchoSet_4.30/epoch=4.ckpt' as top 5

**Comment by JusperLee** [comment]
这个效果是正常的，因为分离模型收敛很慢，所以需要更多的epoch

**Comment by xmxs31117** [comment]
你好，请问预训练权重在哪里发布呢？需要自己从头开始重新训练吗？

**Comment by JusperLee** [comment]
在huggingface上发布了。
https://huggingface.co/JusperLee/TIGER-speech
https://huggingface.co/JusperLee/TIGER-DnR

**Comment by ppeterpp** [comment]
你好，仅提供了large版本的权重吗？请问能否提供small或tiny的权重
